data-raw/standard-format-data-prep/covariate_stock_recruit.Rmd

```diff
@@ -53,10 +88,45 @@ ggplot(butte_dd_sum, aes(wyear, totalddays)) +
 ```
 
 
-## Day of year
+## Day of year when 7DADM drops below 13 C
+#Butte Creek
+```{r}
+#Filtering Butte data 
+butte_std_temp <- read.csv(here::here("data-raw", "standard-format-data-prep", "temp_data", "standard_temperature.csv")) |>
+  filter(stream == "butte creek")
+
+#Changing "date" field into date format
+butte_std_temp$date <- as.Date(butte_std_temp$date)
+glimpse(butte_std_temp)
+
+#Extracting spawning months (Aug-Dec)
+butte_std_temp <- butte_std_temp |> 
+  mutate(month = format(date, "%m"))
```

Comment:
This works! Another option is to use the `lubridate` package. This package is great and you should start to get familiar with it! There is an easy function called `month` so you can do `mutate(month = month(date))`